project_and_code_guidelines.md

```diff
@@ -315,19 +315,20 @@ Example:
 
 ```java
 public class MainActivity extends Activity {
+    private static final String TAG = MainActivity.class.getSimpleName();
```

Comment:
This is personal preference, but I would usually leave an empty line between the declaration of the class and the first line in it. Would you mind moving this constant onto a new line? Then I will be happy to merge it. :)

Reply:
No, I wouldn't mind moving this constant onto a new line :) Just added it in https://github.com/ribot/android-guidelines/pull/30/commits/2b4a6a0c0029a3276995af36466ad1736cc0a9bd commit.

Reply:
Thanks for indulging me! Will merge now. 👍